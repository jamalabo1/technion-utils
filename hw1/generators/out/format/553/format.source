D W 272 1
D X 907 2
A X 862 3
A Y 53 4
C Z 958 5
D Z 365 6
D Y 986 7
A X 988 8
C W 903 9
B W 163 10
A Y 993 11
D W 692 12
D Y 29 13
D Z 493 14
D Y 260 15
C Y 960 16
C X 390 17
D Z 821 18
D Z 335 19
D X 613 20
D Y 390 21
D W 365 22
A W 881 23
B Y 201 24
A X 938 25
B W 90 26
D Z 883 27
C W 100 28
A X 516 29
D Y 393 30
C W 105 31
B X 936 32
B Z 955 33
B W 815 34
C W 158 35
B Z 797 36
C W 965 37
A W 105 38
A W 106 39
D X 911 40
C X 985 41
C W 968 42
C W 81 43
A X 607 44
C X 123 45
A Z 908 46
C X 51 47
B X 834 48
A Z 859 49
D Z 882 50
A Z 551 51
C Y 356 52
B Z 419 53
B W 790 54
A Z 277 55
D Y 447 56
C Z 3 57
D X 209 58
D Z 203 59
D Z 681 60
C W 723 61
D Z 316 62
D Z 350 63
D W 702 64
A X 202 65
C W 10 66
A Y 163 67
B Z 895 68
B W 255 69
D Z 985 70
D Y 596 71
C Z 830 72
C W 399 73
D X 516 74
D Z 349 75
A Y 553 76
B W 946 77
D X 8 78
D Z 9 79
B W 907 80
C Y 827 81
D W 234 82
A Y 559 83
A Z 95 84
B W 869 85
D Y 167 86
B Z 532 87
D X 152 88
A X 376 89
B W 19 90
C W 229 91
B Y 513 92
D W 802 93
A Y 331 94
D W 464 95
D Z 826 96
C Z 446 97
B Y 57 98
C X 390 99
B W 452 100
A X 231 101
B Z 301 102
B X 283 103
C Y 949 104
A X 1 105
C Y 318 106
B W 770 107
B W 598 108
B Z 762 109
B Z 206 110
A Y 80 111
D W 532 112
C W 337 113
D Y 759 114
D Y 597 115
D X 199 116
D W 937 117
A Y 476 118
D Y 540 119
A Z 923 120
B X 586 121
B Z 860 122
B W 528 123
D Y 357 124
C W 249 125
D X 154 126
C Z 108 127
B W 160 128
C Y 317 129
B Y 8 130
D W 810 131
B X 250 132
B X 843 133
B W 67 134
D Y 776 135
D Z 510 136
C Z 10 137
A Y 134 138
D W 869 139
B W 712 140
B X 175 141
A Y 659 142
A Z 854 143
B W 828 144
D Y 549 145
B X 858 146
B W 313 147
C W 395 148
A Z 159 149
A Z 485 150
A X 120 151
D Z 39 152
A X 592 153
D W 530 154
C W 995 155
D X 438 156
B X 963 157
A X 927 158
B Z 895 159
B Z 97 160
D Z 548 161
D Z 230 162
D Y 831 163
D W 318 164
A X 461 165
D X 936 166
C W 846 167
D X 664 168
B W 856 169
A Y 38 170
D X 311 171
C X 719 172
B Y 750 173
D W 403 174
C Z 135 175
A Y 478 176
A Y 993 177
A W 819 178
C Z 983 179
D Y 187 180
D X 907 181
A Y 52 182
C X 823 183
A X 250 184
A Z 628 185
D W 20 186
A X 148 187
D Y 180 188