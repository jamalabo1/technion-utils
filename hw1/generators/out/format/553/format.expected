Blockchain Info:
1
Sender Name: D
Receiver Name: Y
Transaction Value: 180
Transaction timestamp: 188
2
Sender Name: A
Receiver Name: X
Transaction Value: 148
Transaction timestamp: 187
3
Sender Name: D
Receiver Name: W
Transaction Value: 20
Transaction timestamp: 186
4
Sender Name: A
Receiver Name: Z
Transaction Value: 628
Transaction timestamp: 185
5
Sender Name: A
Receiver Name: X
Transaction Value: 250
Transaction timestamp: 184
6
Sender Name: C
Receiver Name: X
Transaction Value: 823
Transaction timestamp: 183
7
Sender Name: A
Receiver Name: Y
Transaction Value: 52
Transaction timestamp: 182
8
Sender Name: D
Receiver Name: X
Transaction Value: 907
Transaction timestamp: 181
9
Sender Name: D
Receiver Name: Y
Transaction Value: 187
Transaction timestamp: 180
10
Sender Name: C
Receiver Name: Z
Transaction Value: 983
Transaction timestamp: 179
11
Sender Name: A
Receiver Name: W
Transaction Value: 819
Transaction timestamp: 178
12
Sender Name: A
Receiver Name: Y
Transaction Value: 993
Transaction timestamp: 177
13
Sender Name: A
Receiver Name: Y
Transaction Value: 478
Transaction timestamp: 176
14
Sender Name: C
Receiver Name: Z
Transaction Value: 135
Transaction timestamp: 175
15
Sender Name: D
Receiver Name: W
Transaction Value: 403
Transaction timestamp: 174
16
Sender Name: B
Receiver Name: Y
Transaction Value: 750
Transaction timestamp: 173
17
Sender Name: C
Receiver Name: X
Transaction Value: 719
Transaction timestamp: 172
18
Sender Name: D
Receiver Name: X
Transaction Value: 311
Transaction timestamp: 171
19
Sender Name: A
Receiver Name: Y
Transaction Value: 38
Transaction timestamp: 170
20
Sender Name: B
Receiver Name: W
Transaction Value: 856
Transaction timestamp: 169
21
Sender Name: D
Receiver Name: X
Transaction Value: 664
Transaction timestamp: 168
22
Sender Name: C
Receiver Name: W
Transaction Value: 846
Transaction timestamp: 167
23
Sender Name: D
Receiver Name: X
Transaction Value: 936
Transaction timestamp: 166
24
Sender Name: A
Receiver Name: X
Transaction Value: 461
Transaction timestamp: 165
25
Sender Name: D
Receiver Name: W
Transaction Value: 318
Transaction timestamp: 164
26
Sender Name: D
Receiver Name: Y
Transaction Value: 831
Transaction timestamp: 163
27
Sender Name: D
Receiver Name: Z
Transaction Value: 230
Transaction timestamp: 162
28
Sender Name: D
Receiver Name: Z
Transaction Value: 548
Transaction timestamp: 161
29
Sender Name: B
Receiver Name: Z
Transaction Value: 97
Transaction timestamp: 160
30
Sender Name: B
Receiver Name: Z
Transaction Value: 895
Transaction timestamp: 159
31
Sender Name: A
Receiver Name: X
Transaction Value: 927
Transaction timestamp: 158
32
Sender Name: B
Receiver Name: X
Transaction Value: 963
Transaction timestamp: 157
33
Sender Name: D
Receiver Name: X
Transaction Value: 438
Transaction timestamp: 156
34
Sender Name: C
Receiver Name: W
Transaction Value: 995
Transaction timestamp: 155
35
Sender Name: D
Receiver Name: W
Transaction Value: 530
Transaction timestamp: 154
36
Sender Name: A
Receiver Name: X
Transaction Value: 592
Transaction timestamp: 153
37
Sender Name: D
Receiver Name: Z
Transaction Value: 39
Transaction timestamp: 152
38
Sender Name: A
Receiver Name: X
Transaction Value: 120
Transaction timestamp: 151
39
Sender Name: A
Receiver Name: Z
Transaction Value: 485
Transaction timestamp: 150
40
Sender Name: A
Receiver Name: Z
Transaction Value: 159
Transaction timestamp: 149
41
Sender Name: C
Receiver Name: W
Transaction Value: 395
Transaction timestamp: 148
42
Sender Name: B
Receiver Name: W
Transaction Value: 313
Transaction timestamp: 147
43
Sender Name: B
Receiver Name: X
Transaction Value: 858
Transaction timestamp: 146
44
Sender Name: D
Receiver Name: Y
Transaction Value: 549
Transaction timestamp: 145
45
Sender Name: B
Receiver Name: W
Transaction Value: 828
Transaction timestamp: 144
46
Sender Name: A
Receiver Name: Z
Transaction Value: 854
Transaction timestamp: 143
47
Sender Name: A
Receiver Name: Y
Transaction Value: 659
Transaction timestamp: 142
48
Sender Name: B
Receiver Name: X
Transaction Value: 175
Transaction timestamp: 141
49
Sender Name: B
Receiver Name: W
Transaction Value: 712
Transaction timestamp: 140
50
Sender Name: D
Receiver Name: W
Transaction Value: 869
Transaction timestamp: 139
51
Sender Name: A
Receiver Name: Y
Transaction Value: 134
Transaction timestamp: 138
52
Sender Name: C
Receiver Name: Z
Transaction Value: 10
Transaction timestamp: 137
53
Sender Name: D
Receiver Name: Z
Transaction Value: 510
Transaction timestamp: 136
54
Sender Name: D
Receiver Name: Y
Transaction Value: 776
Transaction timestamp: 135
55
Sender Name: B
Receiver Name: W
Transaction Value: 67
Transaction timestamp: 134
56
Sender Name: B
Receiver Name: X
Transaction Value: 843
Transaction timestamp: 133
57
Sender Name: B
Receiver Name: X
Transaction Value: 250
Transaction timestamp: 132
58
Sender Name: D
Receiver Name: W
Transaction Value: 810
Transaction timestamp: 131
59
Sender Name: B
Receiver Name: Y
Transaction Value: 8
Transaction timestamp: 130
60
Sender Name: C
Receiver Name: Y
Transaction Value: 317
Transaction timestamp: 129
61
Sender Name: B
Receiver Name: W
Transaction Value: 160
Transaction timestamp: 128
62
Sender Name: C
Receiver Name: Z
Transaction Value: 108
Transaction timestamp: 127
63
Sender Name: D
Receiver Name: X
Transaction Value: 154
Transaction timestamp: 126
64
Sender Name: C
Receiver Name: W
Transaction Value: 249
Transaction timestamp: 125
65
Sender Name: D
Receiver Name: Y
Transaction Value: 357
Transaction timestamp: 124
66
Sender Name: B
Receiver Name: W
Transaction Value: 528
Transaction timestamp: 123
67
Sender Name: B
Receiver Name: Z
Transaction Value: 860
Transaction timestamp: 122
68
Sender Name: B
Receiver Name: X
Transaction Value: 586
Transaction timestamp: 121
69
Sender Name: A
Receiver Name: Z
Transaction Value: 923
Transaction timestamp: 120
70
Sender Name: D
Receiver Name: Y
Transaction Value: 540
Transaction timestamp: 119
71
Sender Name: A
Receiver Name: Y
Transaction Value: 476
Transaction timestamp: 118
72
Sender Name: D
Receiver Name: W
Transaction Value: 937
Transaction timestamp: 117
73
Sender Name: D
Receiver Name: X
Transaction Value: 199
Transaction timestamp: 116
74
Sender Name: D
Receiver Name: Y
Transaction Value: 597
Transaction timestamp: 115
75
Sender Name: D
Receiver Name: Y
Transaction Value: 759
Transaction timestamp: 114
76
Sender Name: C
Receiver Name: W
Transaction Value: 337
Transaction timestamp: 113
77
Sender Name: D
Receiver Name: W
Transaction Value: 532
Transaction timestamp: 112
78
Sender Name: A
Receiver Name: Y
Transaction Value: 80
Transaction timestamp: 111
79
Sender Name: B
Receiver Name: Z
Transaction Value: 206
Transaction timestamp: 110
80
Sender Name: B
Receiver Name: Z
Transaction Value: 762
Transaction timestamp: 109
81
Sender Name: B
Receiver Name: W
Transaction Value: 598
Transaction timestamp: 108
82
Sender Name: B
Receiver Name: W
Transaction Value: 770
Transaction timestamp: 107
83
Sender Name: C
Receiver Name: Y
Transaction Value: 318
Transaction timestamp: 106
84
Sender Name: A
Receiver Name: X
Transaction Value: 1
Transaction timestamp: 105
85
Sender Name: C
Receiver Name: Y
Transaction Value: 949
Transaction timestamp: 104
86
Sender Name: B
Receiver Name: X
Transaction Value: 283
Transaction timestamp: 103
87
Sender Name: B
Receiver Name: Z
Transaction Value: 301
Transaction timestamp: 102
88
Sender Name: A
Receiver Name: X
Transaction Value: 231
Transaction timestamp: 101
89
Sender Name: B
Receiver Name: W
Transaction Value: 452
Transaction timestamp: 100
90
Sender Name: C
Receiver Name: X
Transaction Value: 390
Transaction timestamp: 99
91
Sender Name: B
Receiver Name: Y
Transaction Value: 57
Transaction timestamp: 98
92
Sender Name: C
Receiver Name: Z
Transaction Value: 446
Transaction timestamp: 97
93
Sender Name: D
Receiver Name: Z
Transaction Value: 826
Transaction timestamp: 96
94
Sender Name: D
Receiver Name: W
Transaction Value: 464
Transaction timestamp: 95
95
Sender Name: A
Receiver Name: Y
Transaction Value: 331
Transaction timestamp: 94
96
Sender Name: D
Receiver Name: W
Transaction Value: 802
Transaction timestamp: 93
97
Sender Name: B
Receiver Name: Y
Transaction Value: 513
Transaction timestamp: 92
98
Sender Name: C
Receiver Name: W
Transaction Value: 229
Transaction timestamp: 91
99
Sender Name: B
Receiver Name: W
Transaction Value: 19
Transaction timestamp: 90
100
Sender Name: A
Receiver Name: X
Transaction Value: 376
Transaction timestamp: 89
101
Sender Name: D
Receiver Name: X
Transaction Value: 152
Transaction timestamp: 88
102
Sender Name: B
Receiver Name: Z
Transaction Value: 532
Transaction timestamp: 87
103
Sender Name: D
Receiver Name: Y
Transaction Value: 167
Transaction timestamp: 86
104
Sender Name: B
Receiver Name: W
Transaction Value: 869
Transaction timestamp: 85
105
Sender Name: A
Receiver Name: Z
Transaction Value: 95
Transaction timestamp: 84
106
Sender Name: A
Receiver Name: Y
Transaction Value: 559
Transaction timestamp: 83
107
Sender Name: D
Receiver Name: W
Transaction Value: 234
Transaction timestamp: 82
108
Sender Name: C
Receiver Name: Y
Transaction Value: 827
Transaction timestamp: 81
109
Sender Name: B
Receiver Name: W
Transaction Value: 907
Transaction timestamp: 80
110
Sender Name: D
Receiver Name: Z
Transaction Value: 9
Transaction timestamp: 79
111
Sender Name: D
Receiver Name: X
Transaction Value: 8
Transaction timestamp: 78
112
Sender Name: B
Receiver Name: W
Transaction Value: 946
Transaction timestamp: 77
113
Sender Name: A
Receiver Name: Y
Transaction Value: 553
Transaction timestamp: 76
114
Sender Name: D
Receiver Name: Z
Transaction Value: 349
Transaction timestamp: 75
115
Sender Name: D
Receiver Name: X
Transaction Value: 516
Transaction timestamp: 74
116
Sender Name: C
Receiver Name: W
Transaction Value: 399
Transaction timestamp: 73
117
Sender Name: C
Receiver Name: Z
Transaction Value: 830
Transaction timestamp: 72
118
Sender Name: D
Receiver Name: Y
Transaction Value: 596
Transaction timestamp: 71
119
Sender Name: D
Receiver Name: Z
Transaction Value: 985
Transaction timestamp: 70
120
Sender Name: B
Receiver Name: W
Transaction Value: 255
Transaction timestamp: 69
121
Sender Name: B
Receiver Name: Z
Transaction Value: 895
Transaction timestamp: 68
122
Sender Name: A
Receiver Name: Y
Transaction Value: 163
Transaction timestamp: 67
123
Sender Name: C
Receiver Name: W
Transaction Value: 10
Transaction timestamp: 66
124
Sender Name: A
Receiver Name: X
Transaction Value: 202
Transaction timestamp: 65
125
Sender Name: D
Receiver Name: W
Transaction Value: 702
Transaction timestamp: 64
126
Sender Name: D
Receiver Name: Z
Transaction Value: 350
Transaction timestamp: 63
127
Sender Name: D
Receiver Name: Z
Transaction Value: 316
Transaction timestamp: 62
128
Sender Name: C
Receiver Name: W
Transaction Value: 723
Transaction timestamp: 61
129
Sender Name: D
Receiver Name: Z
Transaction Value: 681
Transaction timestamp: 60
130
Sender Name: D
Receiver Name: Z
Transaction Value: 203
Transaction timestamp: 59
131
Sender Name: D
Receiver Name: X
Transaction Value: 209
Transaction timestamp: 58
132
Sender Name: C
Receiver Name: Z
Transaction Value: 3
Transaction timestamp: 57
133
Sender Name: D
Receiver Name: Y
Transaction Value: 447
Transaction timestamp: 56
134
Sender Name: A
Receiver Name: Z
Transaction Value: 277
Transaction timestamp: 55
135
Sender Name: B
Receiver Name: W
Transaction Value: 790
Transaction timestamp: 54
136
Sender Name: B
Receiver Name: Z
Transaction Value: 419
Transaction timestamp: 53
137
Sender Name: C
Receiver Name: Y
Transaction Value: 356
Transaction timestamp: 52
138
Sender Name: A
Receiver Name: Z
Transaction Value: 551
Transaction timestamp: 51
139
Sender Name: D
Receiver Name: Z
Transaction Value: 882
Transaction timestamp: 50
140
Sender Name: A
Receiver Name: Z
Transaction Value: 859
Transaction timestamp: 49
141
Sender Name: B
Receiver Name: X
Transaction Value: 834
Transaction timestamp: 48
142
Sender Name: C
Receiver Name: X
Transaction Value: 51
Transaction timestamp: 47
143
Sender Name: A
Receiver Name: Z
Transaction Value: 908
Transaction timestamp: 46
144
Sender Name: C
Receiver Name: X
Transaction Value: 123
Transaction timestamp: 45
145
Sender Name: A
Receiver Name: X
Transaction Value: 607
Transaction timestamp: 44
146
Sender Name: C
Receiver Name: W
Transaction Value: 81
Transaction timestamp: 43
147
Sender Name: C
Receiver Name: W
Transaction Value: 968
Transaction timestamp: 42
148
Sender Name: C
Receiver Name: X
Transaction Value: 985
Transaction timestamp: 41
149
Sender Name: D
Receiver Name: X
Transaction Value: 911
Transaction timestamp: 40
150
Sender Name: A
Receiver Name: W
Transaction Value: 106
Transaction timestamp: 39
151
Sender Name: A
Receiver Name: W
Transaction Value: 105
Transaction timestamp: 38
152
Sender Name: C
Receiver Name: W
Transaction Value: 965
Transaction timestamp: 37
153
Sender Name: B
Receiver Name: Z
Transaction Value: 797
Transaction timestamp: 36
154
Sender Name: C
Receiver Name: W
Transaction Value: 158
Transaction timestamp: 35
155
Sender Name: B
Receiver Name: W
Transaction Value: 815
Transaction timestamp: 34
156
Sender Name: B
Receiver Name: Z
Transaction Value: 955
Transaction timestamp: 33
157
Sender Name: B
Receiver Name: X
Transaction Value: 936
Transaction timestamp: 32
158
Sender Name: C
Receiver Name: W
Transaction Value: 105
Transaction timestamp: 31
159
Sender Name: D
Receiver Name: Y
Transaction Value: 393
Transaction timestamp: 30
160
Sender Name: A
Receiver Name: X
Transaction Value: 516
Transaction timestamp: 29
161
Sender Name: C
Receiver Name: W
Transaction Value: 100
Transaction timestamp: 28
162
Sender Name: D
Receiver Name: Z
Transaction Value: 883
Transaction timestamp: 27
163
Sender Name: B
Receiver Name: W
Transaction Value: 90
Transaction timestamp: 26
164
Sender Name: A
Receiver Name: X
Transaction Value: 938
Transaction timestamp: 25
165
Sender Name: B
Receiver Name: Y
Transaction Value: 201
Transaction timestamp: 24
166
Sender Name: A
Receiver Name: W
Transaction Value: 881
Transaction timestamp: 23
167
Sender Name: D
Receiver Name: W
Transaction Value: 365
Transaction timestamp: 22
168
Sender Name: D
Receiver Name: Y
Transaction Value: 390
Transaction timestamp: 21
169
Sender Name: D
Receiver Name: X
Transaction Value: 613
Transaction timestamp: 20
170
Sender Name: D
Receiver Name: Z
Transaction Value: 335
Transaction timestamp: 19
171
Sender Name: D
Receiver Name: Z
Transaction Value: 821
Transaction timestamp: 18
172
Sender Name: C
Receiver Name: X
Transaction Value: 390
Transaction timestamp: 17
173
Sender Name: C
Receiver Name: Y
Transaction Value: 960
Transaction timestamp: 16
174
Sender Name: D
Receiver Name: Y
Transaction Value: 260
Transaction timestamp: 15
175
Sender Name: D
Receiver Name: Z
Transaction Value: 493
Transaction timestamp: 14
176
Sender Name: D
Receiver Name: Y
Transaction Value: 29
Transaction timestamp: 13
177
Sender Name: D
Receiver Name: W
Transaction Value: 692
Transaction timestamp: 12
178
Sender Name: A
Receiver Name: Y
Transaction Value: 993
Transaction timestamp: 11
179
Sender Name: B
Receiver Name: W
Transaction Value: 163
Transaction timestamp: 10
180
Sender Name: C
Receiver Name: W
Transaction Value: 903
Transaction timestamp: 9
181
Sender Name: A
Receiver Name: X
Transaction Value: 988
Transaction timestamp: 8
182
Sender Name: D
Receiver Name: Y
Transaction Value: 986
Transaction timestamp: 7
183
Sender Name: D
Receiver Name: Z
Transaction Value: 365
Transaction timestamp: 6
184
Sender Name: C
Receiver Name: Z
Transaction Value: 958
Transaction timestamp: 5
185
Sender Name: A
Receiver Name: Y
Transaction Value: 53
Transaction timestamp: 4
186
Sender Name: A
Receiver Name: X
Transaction Value: 862
Transaction timestamp: 3
187
Sender Name: D
Receiver Name: X
Transaction Value: 907
Transaction timestamp: 2
188
Sender Name: D
Receiver Name: W
Transaction Value: 272
Transaction timestamp: 1